feature/src/main/feature/feature.xml

```diff
@@ -596,8 +597,9 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.karaf.chef/commands/${project.version}</bundle>
     </feature>
 
-     <feature name="jclouds-url-handler" description="Url Handler for jclouds Blobs" version="${project.version}">
+    <feature name="jclouds-url-handler" description="Url Handler for jclouds Blobs" version="${project.version}">
         <feature version='${project.version}'>jclouds</feature>
+        <feature version='${project.version}'>jclouds-services</feature>
```

Comment:
Ah, I found what happens. 
Given the feature indicates a very old namespace, the karaf plugin disables the verification of runtime service dependencies.  However, if you change the namespace version to `http://karaf.apache.org/xmlns/features/v1.3.0`, then the verification will fail.